src/transactions/feed/TransactionDetailsScreen.tsx

```diff
@@ -104,23 +116,96 @@ function TransactionDetailsScreen({ navigation, route }: Props) {
       break
     case TokenTransactionTypeV2.SwapTransaction:
       content = <SwapContent exchange={transaction as TokenExchange} />
+      retryHandler = () => navigate(Screens.SwapScreenWithBack)
       break
   }
 
+  const transactionNetwork = networkConfig.networkIdToNetwork[transaction.networkId]
+  const openBlockExplorerHandler = transactionNetwork
+    ? () => {
+        navigate(Screens.WebViewScreen, {
+          uri: `${networkConfig.blockExplorerBaseTxUrl[transactionNetwork]}${transaction.transactionHash}`,
+        })
+      }
+    : undefined
+
+  const primaryActionHanlder =
+    transaction.status === TransactionStatus.Failed ? retryHandler : openBlockExplorerHandler
+
   return (
-    <ScrollView contentContainerStyle={styles.contentContainer}>
-      <SafeAreaView style={styles.content}>{content}</SafeAreaView>
```

Comment:
Why removing `SareAreaView`? Doesn't this cause issue scrolling the full content on devices with the home bar at the bottom (iOS)?

Especially when the font size this increased by the user.

Reply:
Oh, I missed that. Thanks for the suggestion! Will bring `SafeAreaView` back.